packages/datadog-plugin-mongodb-core/test/core.spec.js

```diff
@@ -259,30 +175,6 @@ describe('Plugin', () => {
             })
           })
 
-          it('should sanitize the query as the resource', done => {
```

Comment:
This should be kept and simply renamed, since we still want the query as the resource, just not sanitized.

Reply:
Ah, it's testing multiple things, I guess. I'll have to rewrite it to work with non-sanitized queries then...